src/main/scala/swiss/dasch/domain/AssetFilename.scala

```diff
@@ -13,7 +13,7 @@ final case class AssetFilename private (value: String) extends AnyVal
 
 object AssetFilename {
   // Allow letters, numbers, underscores, hyphens, spaces, full stops, comma, single quote, apostrophe and braces
-  private val regex = """^[\p{L}\p{N}_\- .,'`()+=!?<>|]+$""".r
+  private val regex = """^[\p{L}\p{N}_\- .,'`+=!<>|()\[\]\{\}]+$""".r
```

Comment:
I think that `()` need to be escaped as well, because they are reserve characters in regex?

Reply:
They were not escaped before and all tests passed but you might have right.